### PR TITLE
[Issue and PR templates]: Update issue and pull request template

### PR DESCRIPTION
**Pull request by maya**
## Description

Updates issue and pull request templates to reflect the types of issues/PRs that will happen in this repo and provides a link to the web-design-standards repo for Standards changes.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-<!-- Please feel free to remove whatever sections that are not relevant.-->
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant.
 
-## Title line template: Brief statement describing what this pull request solves.
+Use the title line as the title of your pull request, then delete these lines. 
+
+## Title line template: [Title]: Brief description
+
+UI components: For pull requests that impact the look, feel, or functionality of the Standards themselves, please open a pull requests on the web-design-standards repo (https://github.com/18F/web-design-standards-docs).
```
Comment (jameshupp) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should be "please open a pull request on the..." (no `s`).

**Comment by jameshupp** [comment]
Other than the one line edit, this looks good to me!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (maya) [inline .github/PULL_REQUEST_TEMPLATE.md]: Done in 8993132.

**Comment by jameshupp** [comment]
👍

**Pull request merged**